lib/linguist/languages.yml

```diff
@@ -3065,6 +3065,14 @@ MoonScript:
   - moon
   ace_mode: text
   language_id: 238
+Motorola 68K Assembly:
+  type: programming
+  group: Assembly
+  extensions:
+  - ".X68"
+  tm_scope: source.68k
+  ace_mode: assembly_x86
+  language_id: 432
```

Comment:
That language_id looks weird. Did you generate it as per the contribution guidelines?